src/mlpack/methods/ann/activation_functions/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(SOURCES
   rectifier_function.hpp
   softplus_function.hpp
   swish_function.hpp
+  mish_function.hpp
```

Comment:
Hey it looks like mish activation function is still here.

Reply:
Hey @walragatver, I will revert the changes asap. Are the rest of the changes fine?

Reply:
Thank you @walragatver, this really helped in removing changes from my second PR.